core/src/apps/base.py

```diff
@@ -313,17 +301,18 @@ def set_homescreen() -> None:
         set_default(homescreen)
 
 
-def lock_device() -> None:
+def lock_device(interrupt_workflow: bool = True) -> None:
```

Comment:
So this means that a `LockDevice` message will interrupt every workflow. Do we want that? I think yes, but just checking.

Reply:
Actually sending any unexpected message in a workflow will probably interrupt it IIRC, won't it? So the above though is probably irrelevant. I was just trying to understand how this parameter is used. Anyway, it's OK as is.